solr/docker/templates/Dockerfile.official.header.template

```diff
@@ -53,18 +53,18 @@ RUN set -ex; \
   for key in $SOLR_KEYS; do \
     found=''; \
     for server in \
-      ha.pool.sks-keyservers.net \
+      pgp.mit.edu \
+      keyserver.ubuntu.com \
       hkp://keyserver.ubuntu.com:80 \
       hkp://p80.pool.sks-keyservers.net:80 \
```

Comment:
I think sks-keyservers.net domain is dead, so this will not work either?

Reply:
Feel free to remove that one as well! It's last, so I doubt it got any use while I was testing.

Reply:
Done

Reply:
Thanks!